### ISSUE TITLE
Google Colab - can't enable extensions

### ISSUE THREAD

**Issue by batmanscode** [arunnbaba/pandas_ui#13]
I get this output:

![image](https://user-images.githubusercontent.com/29989939/85939703-6230ec00-b935-11ea-8583-88d394030ec8.png)

I can still read files without them but it doesn't really work and I get:

![image](https://user-images.githubusercontent.com/29989939/85939731-8ee50380-b935-11ea-92dc-45f9cdcae8f8.png)

**Comment by arunnbaba** [comment]
I'm sorry, I haven't mentioned in the documention that it doesn't support
on colab.

**Comment by batmanscode** [comment]
Oh I see, that's unfortunate. I'm curious though, why is that?

**Comment by arunnbaba** [comment]
There are some images and HTML files which will saved in your location,
when u execute in colab it won't work properly and other than that,
ipywidgets are not rendered properly in colab. Please do try in your
windows os with chrome.

**Comment by batmanscode** [comment]
Thank you for explaining that. I would assume by extension that it will not work on kaggle as well?

I have tried in my env and unfortunately it still doesn't work :( I've used jupyter in brave so far and since it's built on chromium I think it mostly works the same as chrome in terms of compatibility but I'll give chrome a go anyway and see if that works. If not maybe a reinstall will do the trick.
